docker/nginx/nginx-key.conf

```diff
@@ -1,94 +1,43 @@
 ###
-# Mainflux NGINX Conf
-#
-# Taken for /etc/nginx/nginx.conf on Debian machine
-# and https://github.com/nginxinc/docker-nginx/blob/master/mainline/alpine/nginx.conf
+# Mainflux NGINX Configuration
```

Comment:
Rather add Mainflux copyright header.

We know that this is Ngin conf, redundant comment.